src/main/java/domain/Line.java

```diff
@@ -0,0 +1,69 @@
+package domain;
+
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.Deque;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class Line {
+    private static final String ERROR_SCAFFOLD_EMPTY = "[ERROR] 발판이 생성되지 않았습니다";
+    private static final String ERROR_SCAFFOLD_SEQUENT = "[ERROR] 발판이 연속으로 생성되었습니다";
+    private final List<Scaffold> scaffolds;
+
+    public Line(final Width width, final ScaffoldGenerator scaffoldGenerator) {
+        this.scaffolds = createLine(width, scaffoldGenerator);
+        validateScaffolds(scaffolds);
+    }
+
+    public List<Scaffold> createLine(final Width width, final ScaffoldGenerator scaffoldGenerator) {
+        Deque<Scaffold> scaffolds = new ArrayDeque<>();
+        for (int i = 0; i < width.getValue(); i++) {
+            scaffolds.add(createNonConsistScaffold(scaffoldGenerator, scaffolds));
+        }
+        return new ArrayList<>(scaffolds);
+    }
```

Comment:
1. 정적 팩터리 메서드를 의도하셨다면 아래와같이 구현해보는건 어떨까요?
```suggestion
    private Line(final List<Scaffold> scaffolds) {
        this.scaffolds = scaffolds
        validateScaffolds(scaffolds);
    }

    public static List<Scaffold> create(final Width width, final ScaffoldGenerator scaffoldGenerator) {
        Deque<Scaffold> scaffolds = new ArrayDeque<>();
        for (int i = 0; i < width.getValue(); i++) {
            scaffolds.add(createNonConsistScaffold(scaffoldGenerator, scaffolds));
        }
        return new Line(new ArrayList<>(scaffolds));
    }
```

2. 현재 구조를 유지한다고 하면 `createLine` 메서드는 private 접근제어자로 변경해줘도 괜찮을것같네요 😄